app/views/browse/build-config.html

```diff
@@ -133,10 +133,10 @@
                     <!-- Show the latest build and a chart of the recent build duration and status. -->
                     <div ng-if="builds && (builds | hashSize) > 0" class="build-config-summary">
                       <!-- Show the latest build status first. -->
-                      <div class="h3" style="margin-bottom: 0;">
-                        <span class="latest-build-status">
-                          <status-icon status="latestBuild.status.phase" style="margin-right: 5px;"></status-icon>
-                          Latest build
+                      <div class="h3">
+                        <span class="last-status">
+                          <status-icon status="latestBuild.status.phase"></status-icon>
+                          Last build
```

Comment:
Yeah I was trying to avoid "Latest deployment" since that has a specific meaning (and not always most recent). I changed it here to be consistent, but I can change it back.